### ISSUE TITLE
Issue executing CloudSaga

### ISSUE THREAD

**Issue by Ameurdx** [awslabs/aws-cloudsaga#8]
Hi,

I have a problem executing CloudSaga.

I executed it in CloudShell and I have administrator access to the AWS account.

I am getting this error:

`2022-09-23 07:49:50,878 - INFO - CreateVpc API Call
2022-09-23 07:49:51,775 - ERROR - An error occurred (DryRunOperation) when calling the CreateVpc operation: Request would have succeeded, but DryRun flag is set.
2022-09-23 07:49:51,785 - INFO - Creating rogue Amazon VPC in region ap-southeast-2...
2022-09-23 07:49:51,785 - INFO - CreateVpc API Call
2022-09-23 07:49:52,844 - ERROR - An error occurred (DryRunOperation) when calling the CreateVpc operation: Request would have succeeded, but DryRun flag is set.
2022-09-23 07:49:52,854 - INFO - Creating rogue Amazon VPC in region us-west-2...
2022-09-23 07:49:52,855 - INFO - CreateVpc API Call
2022-09-23 07:49:53,576 - ERROR - An error occurred (DryRunOperation) when calling the CreateVpc operation: Request would have succeeded, but DryRun flag is set.
2022-09-23 07:49:53,620 - INFO - Creating rogue Amazon VPC in region ca-central-1...
2022-09-23 07:49:53,620 - INFO - CreateVpc API Call
2022-09-23 07:49:54,050 - ERROR - An error occurred (DryRunOperation) when calling the CreateVpc operation: Request would have succeeded, but DryRun flag is set.
2022-09-23 07:49:54,058 - INFO - Looking for existing VPCs to run DryRun security events in region eu-central-1...
2022-09-23 07:49:54,059 - INFO - DescribeVpcs API Call
2022-09-23 07:49:54,211 - INFO - Creating Security Group in existing VPC (DryRun)...
2022-09-23 07:49:54,353 - ERROR - An error occurred (DryRunOperation) when calling the CreateSecurityGroup operation: Request would have succeeded, but DryRun flag is set.
2022-09-23 07:49:54,362 - INFO - Looking for existing VPCs to run DryRun security events in region eu-central-1...
2022-09-23 07:49:54,362 - INFO - DescribeVpcs API Call
2022-09-23 07:49:54,423 - INFO - Creating Security Group in existing VPC (DryRun)...
2022-09-23 07:49:54,558 - ERROR - An error occurred (DryRunOperation) when calling the CreateSecurityGroup operation: Request would have succeeded, but DryRun flag is set.
`

`
2022-09-23 07:22:29,268 - INFO - Spinning up Bitcoin Miners (DryRun) in region ap-northeast-3...
2022-09-23 07:22:29,268 - INFO - RunInstances API Call
2022-09-23 07:22:30,334 - ERROR - An error occurred (UnsupportedOperation) when calling the RunInstances operation: Tagging an elastic gpu on create is not yet supported in this region.
2022-09-23 07:22:30,344 - INFO - Spinning up Bitcoin Miners (DryRun) in region us-west-2...
2022-09-23 07:22:30,344 - INFO - RunInstances API Call
2022-09-23 07:22:31,635 - ERROR - An error occurred (DryRunOperation) when calling the RunInstances operation: Request would have succeeded, but DryRun flag is set.
`

`An error occurred (DryRunOperation) when calling the RunInstances operation: Request would have succeeded, but DryRun flag is set
An error occurred (UnsupportedOperation) when calling the RunInstances operation: Tagging an elastic gpu on create is not yet supported in this region
An error occurred (DryRunOperation) when calling the CreateSecurityGroup operation: Request would have succeeded, but DryRun flag is set
`

**Comment by hackersifu** [comment]
The code has the DryRun flag enabled where possible, so that's where those specific error messages are coming from. Here's an example within the code: https://github.com/awslabs/aws-cloudsaga/blob/main/cloudsaga/scenarios/network_changes.py#L53-L55

**Comment by Ameurdx** [comment]
Yes, I get That. With the DryRun Flag nothing will be executed, but the permissions will be tested.
If I want to simulate the security events/ execute the script, how can I deactivate the DryRun Flag?
Thank you

**Comment by za** [comment]
Hi @Ameurdx I think `cloudsaga` is designed to run only with `DryRun` to `True`. It's hardcoded in the code, I am afraid it's not possible to deactivate it. CMIIW @hackersifu .

**Comment by hackersifu** [comment]
Correct @za , you'd have to modify the underlying script to make that change.

**Comment by za** [comment]
In that case, I think we can close this issue @hackersifu cc: @Ameurdx
